docs/02-get-started/01-quick-install.md

```diff
@@ -42,22 +42,12 @@ export CLUSTER_DOMAIN={YOUR_CLUSTER_DOMAIN}
 
 To manage Kyma via GUI, connect it to Kyma Dashboard. 
```

Comment:
```suggestion
To manage Kyma via GUI, open Kyma Dashboard. 
```

You no longer connect Kyma to the the Dashboard, it's already done for you when you run `kyma dashboard`.

Please also adjust the section title by changing it to `## Open Kyma Dashboard`.